README.md

```diff
@@ -29,9 +30,21 @@ In your `apollo.config.js` file:
 
 ```js
 const { config } = require("vscode-apollo-relay").generateConfig()
+// If you want to debug the config, uncomment the following line
+// console.log(JSON.stringify(config))
```

Comment:
I think `JSON.stringify` was introducing all those `nulls in the `validationRules` array. Perhaps something like this would be better? (Untested code.)

```js
console.log(util.inspect(config, false, null, true))
```